gym_roboy/envs/ros_proxy.py

```diff
@@ -7,14 +7,6 @@
 from .msj_robot_state import MsjRobotState
 
 
-class MsjRobotState:
-    def __init__(self, joint_angle, joint_vel):
-        assert len(joint_angle) == MsjROSProxy.DIM_JOINT_ANGLE
-        assert len(joint_vel) == MsjROSProxy.DIM_JOINT_ANGLE
-        self.joint_angle = np.array(joint_angle)
-        self.joint_vel = np.array(joint_vel)
-
```

Comment:
It has nothing to do with merging. Just another thing that you need to point out in PR review